test/OpenTelemetry.Tests/Logs/LogRecordTest.cs

```diff
@@ -30,38 +30,15 @@
 
 namespace OpenTelemetry.Logs.Tests
 {
-    public sealed class LogRecordTest : IDisposable
+    public sealed class LogRecordTest
```

Comment:
The loggerfactory was disposed in IDisposable impln. It does not look like factory is disposed in the new tests.

Reply:
changed the helper method to return the factory. calling method is responsible for using `using`